pkg/streams/streamsDelete.go

```diff
@@ -68,11 +63,11 @@ func deleteStreamFromRestApi(
 					resp,
 					streamName,
 					byteReader,
-					galasaErrors.GALASA_ERROR_GET_STREAMS_NO_RESPONSE_CONTENT,
-					galasaErrors.GALASA_ERROR_GET_STREAMS_RESPONSE_BODY_UNREADABLE,
-					galasaErrors.GALASA_ERROR_GET_STREAMS_UNPARSEABLE_CONTENT,
-					galasaErrors.GALASA_ERROR_GET_STREAMS_SERVER_REPORTED_ERROR,
-					galasaErrors.GALASA_ERROR_GET_STREAMS_EXPLANATION_NOT_JSON,
+					galasaErrors.GALASA_ERROR_DELETE_STREAMS_NO_RESPONSE_CONTENT,
+					galasaErrors.GALASA_ERROR_DELETE_STREAMS_RESPONSE_BODY_UNREADABLE,
+					galasaErrors.GALASA_ERROR_DELETE_STREAMS_UNPARSEABLE_CONTENT,
+					galasaErrors.GALASA_ERROR_DELETE_STREAMS_SERVER_REPORTED_ERROR,
+					galasaErrors.GALASA_ERROR_DELETE_STREAMS_EXPLANATION_NOT_JSON,
```

Comment:
These errors need to accept an identifier in the templates (like `'{}'`) since we're passing in the streamName, otherwise when any of these errors are returned, we'll end up with an extra part when the error gets formatted - for example: "Failed to delete test stream...(EXTRA string='mystream')" is what we'll see at the moment